### DISCUSSION TITLE
Can I discover service's characteristic UUID ?

### DISCUSSION THREAD

**Discussion by andrealmeida98** [hbldh/bleak#444] [Q&A]
OS : Linux
Python: Python3

This is more a question than an issue. 
This code will print the mac and the UUID of the service. Can I use that UUID to discover its characteristics' UUIDs ?

```python
import asyncio
from bleak import BleakScanner


async def run():
    devices = await BleakScanner.discover()
    for d in devices:
            print(d)
            
loop = asyncio.get_event_loop()
loop.run_until_complete(run())

```

**Comment by koenvervloesem** [comment]
I think you're looking for the [service explore example](https://github.com/hbldh/bleak/blob/develop/examples/service_explorer.py).

**Comment by dlech** [comment]
> This is more a question than an issue.

FYI, there is a new [Discussions](https://github.com/hbldh/bleak/discussions) area for this sort of thing.
